src/main/java/org/jitsi/videobridge/stats/VideobridgeStatistics.java

```diff
@@ -95,12 +90,7 @@ public VideobridgeStatistics()
         BundleContext bundleContext
             = StatsManagerBundleActivator.getBundleContext();
 
-        ConfigurationService cfg
-            = ServiceUtils2.getService(bundleContext, ConfigurationService.class);
-        if (cfg != null)
-        {
-            region = cfg.getString(REGION_PNAME, region);
-        }
+        region = Config.region();
```

Comment:
I suggest accessing it as `OctoConfig.Config.region()` since this the config is not specific to `VideobridgeStatistics`

Reply:
Good idea, will change that.

Reply:
Also, the initialization can be moved to the field declaration, it doesn't need to be in the `static` block anymore